guides/common/modules/con_puppet-architecture.adoc

```diff
@@ -0,0 +1,15 @@
+[id="puppet_guide_puppet_architecture_{context}"]
+= Puppet Architecture
+
+Puppet uses a server-client architecture with managed hosts running the Puppet agent and {Project} or any {SmartProxies} acting as a Puppet master.
```

Comment:
This was merged before I got a chance, but Puppet master is a term that's no longer used. Back in Puppet 3 and earlier the server process was called `puppet master` but in Puppet 4 it was deprecated and the new (Java-based) `puppetserver` daemon replaced it. By now the whole migration has pretty much been completed. See https://puppet.com/blog/removing-harmful-terminology-from-our-products/ as well.

Reply:
Sorry @ekohl I didn't want to put pressure on you @maximiliankolb can you take a look at these comments?

Reply:
Thanks a lot for your review @ekohl 
I've addressed all your ideas & concerns in PR #553